Gems/PhysX/Debug/Code/Source/SystemComponent.cpp

```diff
@@ -342,6 +400,168 @@ namespace PhysXDebug
             color.SetB(b);
         }
     }
+
+    void SystemComponent::DrawJointManipulationWindow()
+    {
+        // Check if a joint is selected
+        if (!PhysX::JointRequestBus::HasHandlers(m_selectedJoint))
+        {
+            return;
+        }
+
+        // Find the joint name from cache
+        auto jointIt = m_jointsCache.find(m_selectedJoint);
+        if (jointIt == m_jointsCache.end())
+        {
+            return;
+        }
+
+        const AZStd::string& jointName = jointIt->second;
+        AZStd::string windowTitle = AZStd::string::format("Joint Control: %s###JointManipulation", jointName.c_str());
+
+        if (ImGui::Begin(windowTitle.c_str(), nullptr, ImGuiWindowFlags_AlwaysAutoResize))
+        {
+            ImGui::Text("Joint: %s", jointName.c_str());
+            ImGui::Separator();
+
+            // Joint Status Section - using O3DE API only
+            if (ImGui::CollapsingHeader("Joint Status", ImGuiTreeNodeFlags_DefaultOpen))
+            {
+                // Get current position using O3DE API
+                float position = 0.0f;
+                PhysX::JointRequestBus::EventResult(
+                    position,
+                    m_selectedJoint,
+                    &PhysX::JointRequests::GetPosition);
+                ImGui::Text("Current Position: %.3f", position);
+
+                // Get per-axis positions from transform
+                AZ::Transform jointTransform;
+                PhysX::JointRequestBus::EventResult(
+                    jointTransform,
+                    m_selectedJoint,
+                    &PhysX::JointRequests::GetTransform);
+
+                ImGui::Separator();
+
+                // Get current velocity using O3DE API
+                AZStd::pair<AZ::Vector3, AZ::Vector3> currentVelocity;
+                PhysX::JointRequestBus::EventResult(
+                    currentVelocity,
+                    m_selectedJoint,
+                    &PhysX::JointRequests::GetVelocityGeneral);
+
+                ImGui::Text("Current Velocity:");
+                ImGui::Text("  Linear:  (%.3f, %.3f, %.3f)",
+                    currentVelocity.first.GetX(),
+                    currentVelocity.first.GetY(),
+                    currentVelocity.first.GetZ());
+                ImGui::Text("  Angular: (%.3f, %.3f, %.3f)",
+                    currentVelocity.second.GetX(),
+                    currentVelocity.second.GetY(),
+                    currentVelocity.second.GetZ());
+
+                ImGui::Separator();
+
+                // Get transform using O3DE API
+                PhysX::JointRequestBus::EventResult(
```

Comment:
Declaration removed in my other suggestion for line 439
```suggestion
                AZ::Transform jointTransform;
                PhysX::JointRequestBus::EventResult(
```